content/en/metrics/nested_queries.md

```diff
@@ -0,0 +1,205 @@
+
+{{< img src="/metrics/nested_queries/multilayer-space-agg-json.png" alt="example of multilayer space aggregation in the JSON" style="width:100%;" >}}
+{{% /collapse-content %}} 
+
+
+## Percentiles and Standard Deviation for Aggregated Counts/Rates/Gauges
+
+Percentiles and standard deviation for aggregated counts/rates/gauges are a part of the multilayer aggregation aggregators. They allow you to better understand the variability and spread of your large datasets and allow you to better identify outliers. The percentile/standard deviation calculation offered in nested queries is computed on top of the results of an existing, aggregated metric query. For globally accurate percentiles that are computed on unaggregated, raw values of a metric, use [distribution metrics][9] instead. 
+
+ {{% collapse-content title="Percentiles in Multilayer Time Aggregation example query " level="h5" %}}
+
+We can use percentiles in multilayer time aggregation to additionally summarize the results of our nested query (avg CPU utilization by `env` and `team` every 5 minutes) by calculating the p95th value of this nested query every 30 minutes. 
+
+ {{< img src="/metrics/nested_queries/percentiles-time-agg-ui.png" alt="example of MLA time agg percentiles using nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/percentiles-time-agg-json.png" alt="example of MLA time agg percentiles using nested queries in the JSON" style="width:100%;" >}}
+
+ {{% /collapse-content %}} 
+ 
+
+{{% collapse-content title="Percentiles in Multilayer Space Aggregation example query" level="h5" %}}
+
+We can use percentiles in multilayer space aggregation to additionally summarize the results of our nested query (avg CPU utilization by `env` and `team` every 5 minutes) by calculating the p95th value of this nested query for every unique `env` value. 
+
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/percentiles-space-agg-ui.png" alt="example of MLA space agg percentiles using nested queries in the UI" style="width:100%;" >}}
+
+ {{< img src="/metrics/nested_queries/percentiles-space-agg-json.png" alt="example of MLA space agg percentiles using nested queries in the JSON" style="width:100%;" >}}
+
+  {{% /collapse-content %}} 
+
+
+
+{{% collapse-content title="Standard deviation example query" level="h5" %}}
+
+Standard deviation helps measure the variability or dispersion of a dataset. The following query uses standard deviation with multilayer time aggregation to calculate the standard deviation of our nested query (sum of API request counts, averaged over 4 hour) over longer twelve-hour periods:
+
+In the UI or JSON tab, it would look as follows:
+
+ {{< img src="/metrics/nested_queries/nested-queries-std-ui.png" alt="example of standard deviation with nested queries in the UI" style="width:100%;" >}}
```

Comment:
Can we find another metric query to demonstrate stddev that doesn't look randomly cut off? @ouracheal